website_sale_adj/controllers/main.py

```diff
@@ -1,23 +1,11 @@
 # Copyright 2019 Quartile Limited
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Why switch to AGPL?

Reply:
Sorry, it was my mistake that I mixed up with the old version.